web/packages/teleport/src/services/apps/types.ts

```diff
@@ -51,9 +51,32 @@ export interface App {
   // Integration is the integration name that must be used to access this Application.
   // Only applicable to AWS App Access.
   integration?: string;
+  /** subKind is subKind of an App. */
+  subKind?: AppSubKind;
+  /** permissionSets is a list of AWS IAM Identity Center permission sets
+   * available for this App. The value is only populated if the app SubKind is
+   * aws_ic_account.
+   */
+  permissionSets?: PermissionSet[];
 }
 
 export type UserGroupAndDescription = {
   name: string;
   description: string;
 };
+
+/** AppSubKind defines names of SubKind for App resource. */
```

Comment:
Same here the comment style seems to be inconsistent. 
It they any  official comment styling  recommendation for *.tx file ? 


```
/** AppSubKind defines names of SubKind for App resource. */
export enum AppSubKind {
```

```
// PermissionSet defines an AWS IAM Identity Center permission set that is available
// to an App.
export type PermissionSet = {
```

Reply:
Heh -  `//` is acceptable but `/** */` is recommended for JSDoc. `//` is a leftover from https://github.com/gravitational/teleport/pull/49621/commits/487e12784fd3cdb2587c232c04725a07d19654a8 commit I failed to update. Updated all `//` to `/** */`  in https://github.com/gravitational/teleport/pull/49621/commits/69bd7cbb984ab5eb82751a43d957469059b59b18.